common4j/src/main/com/microsoft/identity/common/java/util/UrlUtil.java

```diff
@@ -43,6 +43,12 @@
 public class UrlUtil {
     private static final String TAG = UrlUtil.class.getSimpleName();
 
+    public static URL appendPathToURL(@NonNull final URL urlToAppend,
+                                      @Nullable final String pathString) // TODO remove post test slice
```

Comment:
This TODO needed? I thought it was for the queryParam. Since it's a util method, you can probably just keep it even after test slice